css-pseudo-classes/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>css-pseudo-classes</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div>
+  <h2>Hover Button</h2>
+  <button>I'm a button!</button>
+</div>
+<div>
+  <h2>Striped Table</h2>
+  <table>
+    <thead class="headings">
+      <th>Student Name</th>
```

Comment:
Don't forget to include a `tr` tag in the `thead` in the future!

Reply:
Got it. Thank you!